### ISSUE TITLE
Issues inserting images. Incorrect media path.

### ISSUE THREAD

**Issue by trapexit** [turnermm/ckgedit#687]
I recently updated a lot of things so forgive me if perhaps I'm missing something. I've got a nginxproxy/nginx-proxy container fronting the linuxserver.io's dokuwiki image with the PHP7 changes I mentioned in a previous ticket. After moving to this I thought this was working but a few things updated recently.

When using ckgedit plugin and attempting to insert an image the URL generated is `/data/media/RELPATH` where RELPATH is the relative media path. For instance: `/data/media/wiki/logo.png`

But the public url to the image is: `/_media/wiki/logo.png` or `/lib/exe/fetch.php?media=wiki:logo.png` when using the DW Filebrowser.

Is the expectation to create a rule to redirect? It looks like this is hard coded into lib/plugins/ckgedit/fckeditor/editor/filemanager/connectors/php/config.php

**Comment by turnermm** [comment]
I have no personal experience with nginx or nginx proxies. But there is a user contributed item on the ckgedit plugin page about how to handle proxies

**Comment by trapexit** [comment]
The question isn't about the proxies but about how paths are generated by the ckgedit file browser. The path generated by ckgedit's file browser is not valid in my setup and I'm trying to find out why. http://domain/data/media/ is not where files are stored on DokuWiki typically.

Take this image from the main dokuwiki site. It uses `/_media/`... https://www.dokuwiki.org/_media/marketing:hackfest.jpg

The result of using ckg file browser
![image](https://user-images.githubusercontent.com/197676/197205980-e025815b-a26f-4bff-85c2-9289b7aa519b.png)

The result of using dw file browser
![image](https://user-images.githubusercontent.com/197676/197206190-2d7a2eee-6767-4792-bc05-8b372bcbdb0b.png)

Whereas the path I would expect to see is `/_media/wiki:logo.png` if relying on rewrites by the webserver.

The code that appears to set this is https://github.com/turnermm/ckgedit/blob/6900a33563e01f3a430460c62479e38b61b97e52/fckeditor/editor/filemanager/connectors/php/config.php#L210 and has been around forever. So in the past either it was modified somewhere or people used rewrite rules but I don't recall setting up rules in a previous setup.

**Comment by turnermm** [comment]
```
_media/wiki/logo.png
 /lib/exe/fetch.php?media=wiki:logo.png
```
These are functionally equivalent.

**Comment by trapexit** [comment]
Yes. But as I showed above the file browser is returning `/data/media/wiki/logo.png`... not `/_media/wiki/logo.png`

**Comment by turnermm** [comment]
`http://domain/data/media/ is not where files are stored on DokuWiki typically.`
Not true.  

In any event the ckgedit filebrowser does get the correct url for its use in the image dialog's location bar:

Here, for instance is an example from a test wiki:
<img width="411" alt="imgurl" src="https://user-images.githubusercontent.com/176447/197308310-012146e4-d129-44c5-8e53-e232c2294e3c.png">

The problem as I initially suggested is probably in the proxy set up.
